src/io/bed.rs

```diff
@@ -8,7 +8,9 @@
 //! # Example
 //!
 //! ```
+//! // import functions (at top of script)
 //! use bio::io::bed;
+//! // run within a function
```

Comment:
I think I'd omit the import and run comments, so that we have a consistent style in our docs
```suggestion
//! use bio::io::bed;
```